README.md

```diff
@@ -48,29 +62,47 @@ server.listen(4000)
 - **WHATWG Fetch API:** the core package depends on [WHATWG Fetch API](https://fetch.spec.whatwg.org/) so it can run and deploy on any environment (Serverless, Workers, Deno, Node).
 - **Easily Extendable:** New GraphQL-Yoga support all [`envelop`](https://www.envelop.dev) plugins.
 
-## Features
+## [Features](https://www.the-guild.dev/graphql/yoga-server/v3)
 
-- [GraphQL over HTTP spec compliant](https://github.com/graphql/graphql-over-http)
-- TypeScript
-- File upload with [GraphQL Multipart Request spec](https://github.com/jaydenseric/graphql-multipart-request-spec)
-- Realtime capabilities
-- Accepts `application/json`, `application/graphql-response+json`, `application/graphql+json`, `application/x-www-form-urlencoded`, `application/graphql` and `multipart/formdata` content-types
+- Fully typed with [TypeScript](https://www.typescriptlang.org)
+- [GraphQL over HTTP spec compliant](https://github.com/enisdenjo/graphql-http/tree/master/implementations/graphql-yoga)
+- [GraphiQL included](https://www.the-guild.dev/graphql/yoga-server/v3/features/graphiql)
+- [File uploads with GraphQL Multipart Request spec](https://www.the-guild.dev/graphql/yoga-server/v3/features/file-uploads)
+- [Subscriptions and realtime capabilities](https://www.the-guild.dev/graphql/yoga-server/v3/features/subscriptions)
+- [Automatic persisted queries](https://www.the-guild.dev/graphql/yoga-server/v3/features/automatic-persisted-queries)
+- [Built-in parsing and validation caching](https://www.the-guild.dev/graphql/yoga-server/v3/features/parsing-and-validation-caching)
+- [Testing utilities](https://www.the-guild.dev/graphql/yoga-server/v3/features/testing)
 - Supports [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
-- Runs everywhere: Can be deployed via `now`, `up`, AWS Lambda, Heroku etc.
+- Runs **everywhere**, including environments like:
+  - [Deno](https://www.the-guild.dev/graphql/yoga-server/v3/integrations/integration-with-deno)
+  - [Bun](https://www.the-guild.dev/graphql/yoga-server/v3/integrations/integration-with-bun)
+  - [Cloudflare Workers](https://www.the-guild.dev/graphql/yoga-server/v3/integrations/integration-with-cloudflare-workers)
+  - [AWS Lambda](https://www.the-guild.dev/graphql/yoga-server/v3/integrations/integration-with-aws-lambda)
+  - [_And other..._](https://www.the-guild.dev/graphql/yoga-server/v3/integrations/z-other-environments)
```

Comment:
nit: and other**s**

Reply:
It's meant as "and other environment**s**", hence singular "other".